k8s/sonarqube/chart/sonarqube/templates/application.yaml

```diff
@@ -28,6 +28,8 @@ spec:
         name: {{ .Release.Name }}-secret
   - name: Application Namespace
     value: {{ .Release.Namespace }}
+  - name: Default cretential for web console
```

Comment:
Could you split this for two fields?

Reply:
Yes, but all bug/suggestion in another branch here only README.md